### ISSUE TITLE
emacs-rtags: broken

### ISSUE THREAD

**Issue by nico202** [NixOS/nixpkgs#24618]
## Issue description
emacs rtags won't install:
```
building path(s) ‘/nix/store/mldb0qbhl8nilrkvpmchgd729c5b5bj1-emacs-rtags-2.0’
unpacking sources
unpacking source archive /nix/store/v28hkqcza1kcx8qd13gqrcf0m9jlcns9-rtags-f85bd60
source root is rtags-f85bd60
patching sources
configuring
configure flags: --prefix=/nix/store/mldb0qbhl8nilrkvpmchgd729c5b5bj1-emacs-rtags-2.0  
/nix/store/zz7zy695h8lrhbqm4p587ixz84bqkzda-stdenv/setup: ./configure: /bin/bash: bad interpreter: No such file or directory
builder for ‘/nix/store/sjrsr2x7w6p60fiff2zr67pi6b6zc76i-emacs-rtags-2.0.drv’ failed with exit code 126
cannot build derivation ‘/nix/store/yv76nsi8qpi7vblcwbfgf6f607l5r7r3-system-path.drv’: 1 dependencies couldn't be built
cannot build derivation ‘/nix/store/bzsd7hhg0mkkcpk2qbnw025sb0dy99y5-nixos-system-lenovo-17.03.882.d1c477e007.drv’: 1 dependencies couldn't be built
error: build of ‘/nix/store/bzsd7hhg0mkkcpk2qbnw025sb0dy99y5-nixos-system-lenovo-17.03.882.d1c477e007.drv’ failed

```
trying to add to configuration.nix  emacs25PackagesNg.rtags

### Steps to reproduce
trying to add to configuration.nix  emacs25PackagesNg.rtags produces the error above

## Technical details

* System: 17.03.882.d1c477e007 (Gorilla)
* Nix version: nix-env (Nix) 1.11.8
* Nixpkgs version: "17.03.882.d1c477e007"

**Comment by matthewbauer** [comment]
This is still a problem, but it looks like it's more rtags not doing the right thing. The [configure script](https://github.com/Andersbakken/rtags/blob/master/configure) looks like:

```
#!/bin/bash

function usage ()
{
    echo "configure [OPTIONS...]"
    echo "  --help|-h                       Display this help"
    echo "  --cmake [arg]                   Use arg instead of \"cmake\""
    echo "  --cmake-argument [arg]          Pass additional arguments to cmake"
    echo "  --cmake-generator [arg]         Use this generator for cmake (e.g. Ninja)"
    echo "  --enable-dev-options            Set certain development options (cpack)"
    echo "  --asan                          Enable -fsanitize=address and -fsanitize=undefined"
    echo "  --debug                         Compile RTags in debug mode"
    echo "  --release                       Compile RTags in release mode (default)"
    echo "  --system-clang                  Don't build llvm/clang as part of RTags' build process (default)"
    echo "  --build-clang                   Download and build llvm/clang as part of RTags' build process"
    echo "  --no-install                    Generate a build that isn't going to be installed"
    echo "  --prefix [arg]                  Set install prefix to arg"
```

/bin/bash should be switch to /bin/sh and it should support "--prefix=P" instead of just "--prefix P"

**Comment by Balletie** [comment]
Should be fixed now by PR #28852. Can you confirm @nico202 and, if it is indeed fixed, can you close this issue?

**Comment by nico202** [comment]
I think it has not reached "unstable" yet, I'll close it in a couple of day
